Chapters/01-zig-weird.qmd

```diff
@@ -97,23 +82,17 @@ info: see `zig build --help` for a menu of options
 
 ### Understanding the project files {#sec-project-files}
 
-After you run the `init` command from the `zig` compiler, some new files
-are created inside of your current directory. First, a "source" (`src`) directory
-is created, containing two files, `main.zig` and `root.zig`. Each `.zig` file
-is a separate Zig module, which is simply a text file that contains some Zig code.
+The 'init' command creates the following structure.
 
+A "source" (`src`) directory with the two text files `main.zig` and `root.zig`. Each file represents a module.
 
-The `main.zig` file for example, contains a `main()` function, which represents
-the entrypoint of your program. It is where the execution of your program begins.
-As you would expect from a C, C++, Rust or Go,
-to build an executabe program in Zig, you also need to declare a `main()` function in your module.
-So, the `main.zig` module represents an executable program written in Zig.
+By convention, `main.zig` is where your main function lives in the case that
+you are building an executable. If you are making a library, the convention

```

Comment:
Nice. I think we can use some parts of these changes.

Reply:
That#s actually from the first lines of the created template.